src/components/TasksTable/index.jsx

```diff
@@ -363,20 +430,60 @@ export default class TasksTable extends Component {
             Reset
           </Button>
         </div>
-        <div className={classes.tableWrapper}>
-          <DataTable
-            items={data}
-            renderRow={item => (
-              <TableRow tabIndex={-1} key={item.summary}>
+        <DataTable
+          columns={[
+            {
+              width: Math.max((windowWidth / 2000) * 180, 100),
```

Comment:
Can you explain how you came with these numbers. I want to understand them :)

Reply:
Mostly by just inspect elementing the old table, and try to find numbers that match the cell proportions.